src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/GoogleServicesDAO.scala

```diff
@@ -45,6 +45,9 @@ trait GoogleServicesDAO extends ReportsSubsystemStatus {
 
   def trialBillingManagerRemoveBillingAccount(projectName: String, targetUserEmail: String): Boolean
 
+  def checkGoogleGroupExists(groupEmail: String): Boolean // Option[String]
```

Comment:
is the comment here leftover code from development?

Reply:
yes, thanks, will fix!